packages/serverpod/lib/src/database/database_connection.dart

```diff
@@ -737,5 +746,5 @@ abstract class IncludeList extends Include {
   List<Order>? orderByList;
 
   /// The nested includes
-  Include? include;
+  T? include;
```

Comment:
Question: Can this ever be anything other than an `IncludeObject`?

Reply:
This is so that we can get the real type in the generated code later!

Reply:
Not sure I understand. Doesn't both `IncludeObject` and `IncludeList` inherit from `Include`?

Reply:
Should be able to type this directly to IncludeObject